sdktests/server_side_stream_validation.go

```diff
@@ -0,0 +1,129 @@
+package sdktests
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/launchdarkly/go-test-helpers/v2/httphelpers"
+	"github.com/launchdarkly/go-test-helpers/v2/jsonhelpers"
+	m "github.com/launchdarkly/go-test-helpers/v2/matchers"
+	"github.com/launchdarkly/sdk-test-harness/framework/ldtest"
+	"github.com/launchdarkly/sdk-test-harness/mockld"
+	"github.com/launchdarkly/sdk-test-harness/servicedef"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
+	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
+)
+
+func doServerSideStreamValidationTests(t *ldtest.T) {
```

Comment:
The comment in the previous file also applies here. Integration-harness can't simulate any of these conditions because there's no way to cause a real LD stream to emit these kinds of unexpected data; the SDKs have to be able to handle them in case a stream gets garbled by network problems, gremlins, etc.